lib/pages/timetable/view/timetable_page.dart

```diff
@@ -66,7 +66,22 @@ class _TimetablePageState extends State<TimetablePage> {
       needsToBeAuthenticated: true,
       leading: AppScaffoldAction(
         icon: Icons.today_outlined,
-        onPressed: () => _controller.animateToToday(),
+        onPressed: () =>
+            !_controller.currentlyVisibleDates.contains(LocalDate.today())
+                ? _controller.animateToToday()
+                : ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
We're using toasts in the app, we don't have snackbars anywhere else so I'd say we shouldn't use a new visual language just for this.

For example usages, look for `AppToast.show` in the codebase.